pages/dkp/kommander/2.1/workspaces/applications/catalog-applications/dkp-applications/spark-operator/index.md

```diff
@@ -16,6 +16,8 @@ The Kubernetes Operator for Apache Spark aims to make specifying and running Spa
 
 You can find generic installation instructions for workspace catalog applications on the [Application Deployment topic](../../application-deployment).
 
+<p class="message--note"><strong>NOTE: </strong>You should only install the Spark operator once per workspace.</p>
```

Comment:
```suggestion
<p class="message--note"><strong>NOTE: </strong>Only install the Spark operator once per workspace.</p>
```

Reply:
fixed in 7195cc0c82201695a22e0befdc2b61ff9f2661c8